EnglishFilter/sections/general_extensions.txt

```diff
@@ -252,6 +252,8 @@ kat.*,kickass.*,kickass2.*,kickasstorrents.*,kat2.*,kattracker.*,thekat.*,thekic
 !
 !
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/101390
+hakaraw.com#%#//scriptlet("abort-current-inline-script", "window.onload", "localStorage.puTS")
```

Comment:
I can't find it on my end. Do you still have it? If so, please show this script in the screenshot.

Reply:
It seems that the advertisement script is blocked by Yuki2718 at https://github.com/AdguardTeam/AdguardFilters/commit/c94523c7e9bc406ded4d69d382fcddbe97c5d19a.
I close this PR.